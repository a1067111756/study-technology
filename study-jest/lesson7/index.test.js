/* 快照测试例子 */
const { generateConfig, generateDynamicConfig } = jest.requireActual("./index");

test("快照测试基本例子", () => {
  expect(generateConfig()).toMatchSnapshot();
});

test("快照测试含有动态属性的例子", () => {
  expect(generateDynamicConfig()).toMatchSnapshot({
    data: expect.any(Date),
  });
});

test("行内快照测试例子", () => {
  expect(generateConfig()).toMatchInlineSnapshot(`
    Object {
      "domain": "www.baidu.com",
      "port": 3000,
      "url": "sicascia.cn",
    }
  `);
});
