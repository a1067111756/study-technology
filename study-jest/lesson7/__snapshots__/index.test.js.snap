// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试含有动态属性的例子 1`] = `
Object {
  "data": Any<Date>,
  "domain": "www.baidu.com",
  "port": 3000,
  "url": "sicascia.cn",
}
`;

exports[`快照测试基本例子 1`] = `
Object {
  "domain": "www.baidu.com",
  "port": 3000,
  "url": "sicascia.cn",
}
`;
